src/app/shared/plnkr/plnkr.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { Http } from '@angular/http';
+import { PlunkerUtil } from './plnkr-util';
+
+@Component({
+  selector: 'app-plnkr',
```

Comment:
Maybe "plunker-example"? (also be consistent with plnkr vs plunker, my vote is for everything to be "plunker")